packages/flutter_tools/lib/src/cache.dart

```diff
@@ -99,12 +97,15 @@ class DevelopmentArtifact {
 }
 
 /// A wrapper around the `bin/cache/` directory.
+///
+/// This does not provide any artifacts by default. See [FlutterCache] for the default
```

Comment:
Can this be made abstract then?

Reply:
No, because it is constructed for the g3 roller with its own artifacts. We could make the FluterCache support that, but then it sort of defeats the purpose of default configuration.